app/controllers/case_management/tax_returns_controller.rb

```diff
@@ -23,17 +23,93 @@ def update
       redirect_to case_management_clients_path, notice: success_message
     end
 
-    def edit_status; end
+    def edit_status
+      # prefill status if we received a param
+      preselected_status = params[:tax_return][:status] if params[:tax_return] && params[:tax_return][:status]
```

Comment:
Consider `.dig` here:

```
      preselected_status = param.dig(:tax_return, :status)
```

Given that simplicity, consider inlining it below.

Not a blocker.

Reply:
Thanks! For some reason I forgot about dig